compiler/bindings/python/CMakeLists.txt

```diff
@@ -128,7 +128,7 @@ declare_mlir_python_sources(IREECompilerAPIPythonTools
     tools/tflite.py
     tools/import_onnx/__main__.py
     tools/ir_tool/__main__.py
-    tools/scripts/ireec/__main__.py
+    tools/scripts/iree_compile/__main__.py
```

Comment:
I thought we had no uses of the `ireec` console script naming, but https://github.com/nod-ai/sharktank/blob/2fecbfdac3a2f9b819b378d4c925cad2cd77532a/libshortfin/examples/python/mobilenet_server/build_model.sh#L42-L43 was recently added. @stellaraccident do you know of other locations to update too?

Reply:
I don't know of others but it's been there for a very long time. Pretty niche, though. Feel free to drop it and we can adapt.